modules/ROOT/pages/access-source-message-attributes.adoc

```diff
@@ -0,0 +1,146 @@
+= Accessing Source Message Attributes in Translation Maps
+
+*Vijay, I am finding the text describing the use cases awkward when I use the term "accessing." To me, you are adding source attributes to translation maps or maybe exposing them, but not really accessing them. If you agree with "Exposing Source Message Attributes in Translation Maps,"" I will change the use case wording accordingly.* 
+
+When you create translation maps for inbound or outbound message flows, you can access source message attributes in those maps. The source attributes in the maps appear in the translated message payload, and you can view them on the activity screen for transmissions that use the map. 
```

Comment:
Please change as follows:
When you create translation or reference maps for inbound or outbound message flows, you can access source message attributes in those maps. You can then map the attributes out in the translated message payload, or use the attribute values in your mapping logic to perform dynamic condition execution.

Reply:
OK - removed "out" though